core/css.js

```diff
@@ -957,4 +957,4 @@ Blockly.Css.CONTENT = [
   '}',
 
   ''
-];
```

Comment:
@ramiAbdou This line is still showing as having the final line ending removed. Can you check your code editor's settings to make sure that it will retain the trailing newline? It might be removing it based on a configuration option.

Reply:
Should be solved now.